### ISSUE TITLE
Example case of task_arithmetic needed

### ISSUE THREAD

**Issue by Opdoop** [arcee-ai/mergekit#392]
Parameter setting in examples is too simple. It's really hard to follow how to set parameters for different methods. For example, `task_arthmetic` is missing. How to merge different layers with different weights in `task_arthmetic`.

**Comment by jukofyork** [comment]
If it's any help then I the way I did it was to use `task_arthmetic` and then afterwards `linear` to filter the layers / tensor names:

https://huggingface.co/jukofyork/miquplus-midnight-70b
